src/Storageless/Http/SessionMiddleware.php

```diff
@@ -221,7 +221,7 @@ private function parseToken(Request $request)
      *
      * @return SessionInterface
```

Comment:
It can be `private` since it's being called from a closure... [works fine even on PHP 5.6](https://3v4l.org/j68oI)